### DISCUSSION TITLE
Text embeddings other than text-embedding-ada-002

### DISCUSSION THREAD

**Discussion by ghost** [crewAIInc/crewAI#1465] [General]
I want to use text embedding model "text-embedding-3-large" how can i use it? i even specified it in DirectorySearchTool but it is not working.

**Comment by zinyando** [comment]
Can you share your DirectorySearchTool configuration?

**Comment by ghost** [comment]
Hi, i have been facing problems when creating directory search tool. this is my code """tool = DirectorySearchTool(directory="policies")""" and the problem is it does not creates any db or text embeddings. the process gets finished at 0. Even while using the default text embedding it is not working as expected
<img width="794" alt="Screenshot 2024-10-18 at 9 59 23 AM" src="https://github.com/user-attachments/assets/8925f3fd-319f-46a3-a3fc-43a54481461d">
